exercise/change_attributes.py

```diff
@@ -0,0 +1,147 @@
+"""
+This module contains a code for exercises 17-1 related to:
+Think Python, 2nd Edition
+Chapter 17: Classes and Methods
+
+Modify the methods (and the function int_to_time) to work with the
+new implementation.
+"""
+
+class Time(object):
+    """
+    Represents the time of day.
+    """
+
+    def __init__(self, hour=0, minute=0, second=0):
+        """
+        Initializes a Time object.
+
+        Attributes: hour, minute, second
+        """
+        
+        minutes = hour * 60 + minute
+        self.int_time = (minutes * 60) + second
```

Comment:
Can you get another name for this variable?

Reply:
Done and remove ()